.github/workflows/main.yml

```diff
@@ -160,6 +160,11 @@ jobs:
             issues.txt
       - name: Compare issues
         run: git diff --no-index --word-diff-regex=. --color=always explcheck/testfiles/${{ matrix.release }}-issues.txt issues.txt
+      - name: Check if the configuration is minimal
+        run: |
+          set -e
+          export LUAINPUTS=explcheck/src
+          explcheck/src/prune-explcheck-config.lua explcheck/testfiles/${{ matrix.release }}-issues.txt
```

Comment:
This has limited usefulness, since the maintainer has to manually check the recommendations for all TeX Live releases. A script named `prune-explcheck-config.sh` should also be added that will use `gh` to collect all recommendations (can be matched with the regex `can be removed\.$`) from a CI run and print only those that apply to _all_ TeX Live releases.

We can also try adding `prune-explcheck-config.sh` in a separate CI step that will fail if any recommendations are printed. This will force the maintainer to act and keep the configuration up-to-date.

Reply:
Fixed in commit 978baaf896481c683813b5a897946e12ddb7ee5b from this PR.